sympy/printing/tests/test_tensorflow.py

```diff
@@ -129,6 +161,8 @@ def test_tensorflow_printing():
             " [tensorflow.math.exp(z), -t]])"
 
 
+# The XFAIL is added because of the PR: https://github.com/sympy/sympy/issues/18090
+@XFAIL
```

Comment:
I think it would be better to write
```
# This (random) test is XFAIL because it fails occasionally
# See https://github.com/sympy/sympy/issues/18090
```

Reply:
I think it would be better to refer to the issue (#18090) than this PR (#18469)